assistants/models.tsp

```diff
@@ -9,13 +9,41 @@ using TypeSpec.OpenAPI;
 
 namespace OpenAI;
 
+/**
+ * Specifies the format that the model must output. Compatible with [GPT-4
+ * Turbo](/docs/models/gpt-4-and-gpt-4-turbo) and all GPT-3.5 Turbo models since
+ * `gpt-3.5-turbo-1106`.
+ * 
+ * Setting to `{ "type": "json_object" }` enables JSON mode, which guarantees the message the model
+ * generates is valid JSON.
+ * 
+ * **Important:** when using JSON mode, you **must** also instruct the model to produce JSON
+ * yourself via a system or user message. Without this, the model may generate an unending stream
+ * of whitespace until the generation reaches the token limit, resulting in a long-running and
+ * seemingly "stuck" request. Also note that the message content may be partially cut off if
+ * `finish_reason="length"`, which indicates the generation exceeded `max_tokens` or the
+ * conversation exceeded the max context length.
+ */
+alias AssistantsApiResponseFormatOption = "none" | "auto" | AssistantsApiResponseFormat;
```

Comment:
```suggestion
@extension("x-oaiExpandable", true)
union AssistantsApiResponseFormatOption {
  "none",
  "auto",
  AssistantsApiResponseFormat
}
```

We need to turn this into an union if we want to tag it with oaiExpandable
![image](https://github.com/joseharriaga/openai-in-typespec/assets/21109913/26b59558-b21e-4120-b6b7-d9705b206aa2)